src/components/Clock.js

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable react/button-has-type */
+/* eslint-disable consistent-return */
+
+import React from 'react';
+import './Clock.scss';
+
+export class Clock extends React.Component {
+  state = {
+    date: new Date(),
+    name: '$',
+    visibility: false,
+    previousName: '1',
+  };
+
+  constructor() {
+    super();
+    this.handleClickClock = this.handleClickClock.bind(this);
+    this.handleClickName = this.handleClickName.bind(this);
+  }
+
+  componentDidMount() {
+    setInterval(() => {
+      this.setState({ date: new Date() });
+      // eslint-disable-next-line
+    console.log(this.state.date.toLocaleTimeString());
+    }, 1000);
+  }
+
+  handleClickClock() {
```

Comment:
```suggestion
  toggleClockVisibility() {
```
add meaningful name to the method